statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1558,6 +1565,34 @@ def fit_constrained(self, constraints, start_params=None, **fit_kwds):
         res._results.results_constrained = res_constr
         return res
 
+    @property
+    def offset_name(self):
```

Comment:
I know offset is delattr'ed if it is None, but a property can't be delattr'ed so keeping it around to just return None.